docs/cloud/features/security/security.md

```diff
@@ -0,0 +1,70 @@
+# Security Overview
+
+
+At Tobiko, we treat security as a first-class citizen because we know how valuable your data assets are. Our team follows and executes security best practices across each layer of our product. 
+
+## Tobiko Cloud Standard Deployment
+
+Our standard Tobiko Cloud deployment consists of several components that are each responsible for different parts of the product. 
+
+Below is a diagram of the components along with their descriptions. 
+
+![tobiko_cloud_standard_deployment](./tcloud_standard_deployment.png){ width=80% height=60% style="display: block; margin: 0 auto" }
+
+- **Scheduler**: Orchestrates schedule cadence and hosts state metadata(code versions, logs, cost)
+- **Executor**: Applies code changes and runs SQL queries (actual data processing in SQL Engine) and python models in proper DAG order.
+- **Gateway**: Stores authentication to SQL Engine. Secured through encryption.
+- **SQL Engine**: Processes and stores data based on the above instructions within the **customer’s** environment.
+
+## Tobiko Cloud Hybrid Deployment
+
+For some customers, our hybrid deployment option is a great fit. It provides a seamless experience with Tobiko Cloud but within your own VPC and infrastructure.  
+
+In a hybrid deployment, Tobiko Cloud does not execute tasks directly with the engine. Instead, it passes tasks to the executors hosted in your environment, which then execute the tasks with the engine. 
+
+Executors are Docker containers that connect to both Tobiko Cloud and your SQL engine. They pull work tasks from the Tobiko Cloud scheduler and execute them with your SQL engine. This is a pull-only mechanism authenticated through an OAuth Client ID/Secret. Whitelist IPs in your network to allow reaching Tobiko Cloud IPs from the executor: 34.28.17.91, 34.136.27.153, 34.136.131.20
+
+Below is a diagram of the components along with their description. 
```

Comment:
You need duplicate and update it for the nuances. See the slide I sent in this PR as a comment.

- **Scheduler**: Orchestrates schedule cadence and hosts state metadata(code versions, logs, cost)
- **Executor**: Applies code changes and runs SQL queries (actual data processing in SQL Engine) and python models in proper DAG order.
- **Gateway**: Stores authentication to SQL Engine. Secured through encryption.
- **SQL Engine**: Processes and stores data based on the above instructions within the **customer’s** environment.